application.py

```diff
@@ -20,20 +22,32 @@ def index():
             file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
             filesFound = {'file':open(app.config['UPLOAD_FOLDER'] + filename, 'rb')}
```

Comment:
Use 

```
with open("x.txt") as f:
```

Currently the delete does not work because you still have the file open when trying to delete.

Reply:
+1